src/mbgl/renderer/image_manager.cpp

```diff
@@ -201,7 +201,9 @@ std::set<std::string> ImageManager::getAvailableImages() const {
     MLN_TRACE_FUNC()
     std::lock_guard<std::recursive_mutex> readWriteLock(rwLock);
 
-    MLN_TRACE_ZONE(copy)
+    {
+        MLN_TRACE_ZONE(copy)
```

Comment:
Don't you need to include the return? Otherwise this zone does not measure anything, does it?

Reply:
I am not sure how Tracy zone works, so why the return needs to be included?

Reply:
I think it uses RAII to check how long something is executing. So this will always report 0ms.

Reply:
Then we cannot use multiple ZoneScoped in the same scope.